vars/checkoutQaInternal.groovy

```diff
@@ -4,6 +4,6 @@ def call(Map params = [:]) {
     dir("scylla-qa-internal") {
         git(url: 'git@github.com:scylladb/scylla-qa-internal.git',
             credentialsId:'b8a774da-0e46-4c91-9f74-09caebaea261',
-            branch: 'master')
+            branch: 'find-stress-command')
```

Comment:
you still have things needed to be merge into `scylla-qa-internal` ? please open PRs for them